src/web3/utils.ts

```diff
@@ -93,3 +95,10 @@ export function buildTxo(kit: ContractKit, tx: CeloTx): CeloTxObject<never> {
     },
   }
 }
+
+export function getNetworkFromNetworkId(networkId?: NetworkId): Network | undefined {
```

Comment:
Hm, not entirely sure I understand. We still need the env-agnostic concept of `Network` throughout the app, since certain features (like these sends) care about the actual blockchain, not whether or not we're on test/mainnet. Without using `Network`, we'd need to explicitly check e.g. `if(networkId === NetworkId['ethereum-mainnet'] || networkId === NetworkId['ethereum-sepolia'])`.